html/interactive-guides/circuit-breaker/circuit-breaker-welcome.html

```diff
@@ -21,11 +21,20 @@
     <div class="bankHeadingBlock">
         <div class="bankHeading">Global eBank</div>
```

Comment:
Hey.....did you forget to stick the icons in here?  I don't see the bank-info and the bank-user showing up when I test in this div!

Reply:
Sorry, I thought I dropped all the changes but I didn't.